components/ChooseFourPics.js

```diff
@@ -21,18 +21,18 @@ export default function ChooseFourPics() {
           height="400"
         />
       </StyledA>
-      <StyledA href="https://i.imgur.com/SglfvOg.jpeg/">
+      <StyledA href="https://i.imgur.com/rTEFFzZ.webp?maxwidth=1520&fidelity=grand/">
         <FourPics
```

Comment:
this name suggests that 4 images are being displayed per component, while in reality this only displays one image and is used four times.
I would rename the component to something like `StyledImage` to be more general